selfdrive/boardd/boardd.cc

```diff
@@ -353,7 +353,7 @@ void can_health(PubSocket *publisher) {
   }
 
 #ifndef __x86_64__
-  if ((no_ignition_cnt > NO_IGNITION_CNT_MAX) && (health.usb_power_mode == (uint8_t)(cereal::HealthData::UsbPowerMode::CDP))) {
+  if ((no_ignition_cnt > NO_IGNITION_CNT_MAX) && (health.usb_power_mode == (uint8_t)(cereal::HealthData::UsbPowerMode::CDP)) || (health.voltage < 10500)) {
```

Comment:
can you add a `&& !ignition` to the whole condition? need to make sure we don't switch usb mode while car is on, since I think it causes a brief USB line disconneciton.

Reply:
Also, need to adjust the offroad alert (`selfdrive/controls/lib/alerts_offroad.json`) to mention that low voltage can be the cause of EON not charging.

Reply:
"EON charging disabled after car being off for more than 3 days **or car voltage too low**..."

Reply:
Done!